### DISCUSSION TITLE
Problem where scheduled tasks will be executed repeatedly when starting with multiple workders

### DISCUSSION THREAD

**Discussion by yAoOw** [fastapi/fastapi#12012] [Questions]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

i use this command to start my application 
`
nohup uvicorn app:app --host 0.0.0.0 --port 8000 --workers 5 > ../app.log 2>&1 &`
my task code 
`async def my_task():
      do some curd code 

trigger = CronTrigger(hour=18, minute=35)

scheduler.add_job(my_task, trigger)

scheduler.start()
`

This task will be executed repeatedly, resulting in duplicate data

**Comment by YuriiMotov** [comment]
Please, provide [minimal reproducible example](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://en.wikipedia.org/wiki/Minimal_reproducible_example&ved=2ahUKEwiXhpSsnfqHAxXO_7sIHQo6AwoQFnoECBMQAw&usg=AOvVaw399odp-jUSwdDkWG0KIqN6)

**Comment by YuriiMotov** [comment]
Closing this. Feel free to reopen it when you are ready to provide more info
